setup.py

```diff
@@ -2,7 +2,7 @@
 
 if __name__ == '__main__':
     setup(
-        version='1.0.2',
+        version='1.0.3',
```

Comment:
```suggestion
        version='1.1.0',
```

semver 1.1.0 for new feature.